src/lib.rs

```diff
@@ -3133,6 +3139,34 @@ impl Build {
         }
     }
 
+    fn getenv_with_target_prefixes(&self, var_base: &str) -> Result<String, Error> {
```

Comment:
These functions have been moved and renamed to be grouped with other getenv family of functions, with no other changes intended.